### PR TITLE
Small changes to exp2 ceiling and SLP output

### LINKED ISSUE #626: Tiny change to GSL experimental ceiling diagnostic
This is a one-line change to GSL's experimental ceiling diagnostic #2, for an FAA deliverable.  

This update will also correct the GRIB2 name for the MAPS SLP approach to make it consistent with RAP/HRRR.  This variable is no longer output in RRFS, but just for consistency and future use.

### PR DESCRIPTION
**Pull request by EricJames-NOAA**
This PR includes a small revision to the RAP/HRRR exp2 ceiling diagnostic to improve performance for the RRFS.  It also removes MLSMA (MAPS SLP reduction) output, based on conversations with Jacob Carley and Andrew Benjamin (NCEP) and Curtis Alexander (GSL).  It also corrects the GRIB2 name for the MAPS SLP (should be MSLMA instead of PRMSL based on https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-3.shtml) in post_avblflds.xml.

The changes were tested for RRFS_B (CONUS 3km) on Jet; the only change to output was removal of the MAPS SLP, and changes in the exp2 ceiling field.

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
@EricJames-NOAA Could you sync your branch with the latest commit of the develop branch from the authoritative UPP repos.?

**Comment by WenMeng-NOAA** [comment]
@EdwardColon-NOAA Please let me know if you have comments on grib2 name change of MAPS SLP reduction in 3drtma control file.

**Review by EdwardColon-NOAA — APPROVED** [review]

**Comment by EdwardColon-NOAA** [comment]
The change in the ceiling diagnostic seems reasonable. As long as the update is reflected in the 3drtma control file, there should not be a downstream issue with the analysis.

**Comment by WenMeng-NOAA** [comment]
@EricJames-NOAA Could you add change logs in the source code you modify?

File: parm/post_avblflds.xml [hunk 1]
```diff
@@ -3883,7 +3883,7 @@
     <param>
        <post_avblfldidx>445</post_avblfldidx>
        <shortname>MAPS_PRMSL_ON_MEAN_SEA_LVL</shortname>
-       <pname>PRMSL</pname>
+       <pname>MSLMA</pname>
```
Comment (WenMeng-NOAA) [inline parm/post_avblflds.xml:3886]: @EricJames-NOAA MSLMA is a ncep local use grib2 name. You may add a line "<table_info>NCEP</table_info>" after line 3886.

File: parm/postxconfig-NT-3drtma.txt [hunk 2]
```diff
@@ -356,7 +356,7 @@ MAPS_PRMSL_ON_MEAN_SEA_LVL
 ?
 1
 tmpl4_0
-PRMSL
+MSLMA
 ?
```
Comment (WenMeng-NOAA) [inline parm/postxconfig-NT-3drtma.txt]: @EricJames-NOAA Please change line 360  "?" into "NCEP" for writing MSLMA in 3drtma products.

**Comment by WenMeng-NOAA** [comment]
There are following changed results from RT tests from hrrr/rap:
```
rap:
[Wen.Meng@hfe04 ~/stmp2]$ cat rap_2020072316/WRFNAT.GrbF16.diff
1070:282595626:HGT:cloud base:rpn_corr=0.808426:rpn_rms=5241.32
1071:283798237:PRES:cloud base:rpn_corr=0.830284:rpn_rms=24110.9

hrrr:
[Wen.Meng@hfe04 ~/stmp2]$ cat hrrr_2020060118/WRFTWO.GrbF04.diff
121:85620587:HGT:cloud base:rpn_corr=0.896319:rpn_rms=3621.46
122:88016324:PRES:cloud base:rpn_corr=0.895746:rpn_rms=15131.5
```
@EricJames-NOAA Can you confirm the above changes are expected? Thanks!

**Comment by WenMeng-NOAA** [comment]
The grib2 name of  MAPS SLP reduction in 3drtma will be changed from PRMSL to MSLMA.

**Comment by EricJames-NOAA** [comment]
> There are following changed results from RT tests from hrrr/rap:
> 
> ```
> rap:
> [Wen.Meng@hfe04 ~/stmp2]$ cat rap_2020072316/WRFNAT.GrbF16.diff
> 1070:282595626:HGT:cloud base:rpn_corr=0.808426:rpn_rms=5241.32
> 1071:283798237:PRES:cloud base:rpn_corr=0.830284:rpn_rms=24110.9
> 
> hrrr:
> [Wen.Meng@hfe04 ~/stmp2]$ cat hrrr_2020060118/WRFTWO.GrbF04.diff
> 121:85620587:HGT:cloud base:rpn_corr=0.896319:rpn_rms=3621.46
> 122:88016324:PRES:cloud base:rpn_corr=0.895746:rpn_rms=15131.5
> ```
> 
> @EricJames-NOAA Can you confirm the above changes are expected? Thanks!

@WenMeng-NOAA yes, that's consistent with my tests!  The GSL exp2 ceiling diagnostic is labeled HGT/PRES on cloud base according to post_avblflds_raphrrr.xml.  That is the only field I expect to change from this change.  Thanks.

**Comment by WenMeng-NOAA** [comment]
@EricJames-NOAA Can you update post_avblflds.xml based on my comments and sync your branch with latest commit from the develop branch? Thanks!

**Comment by EricJames-NOAA** [comment]
@WenMeng-NOAA I think I just synced with the latest commit from the develop branch.  I already added the NCEP table info line for post_avblflds.xml...please let me know if there are other changes needed for this file.

**Comment by EricJames-NOAA** [comment]
@WenMeng-NOAA looks like one of the checks failed...maybe I did something wrong when syncing my branch with the latest EMC develop...let me know what you recommend...

**Comment by WenMeng-NOAA** [comment]
> @WenMeng-NOAA I think I just synced with the latest commit from the develop branch. I already added the NCEP table info line for post_avblflds.xml...please let me know if there are other changes needed for this file.

@EricJames-NOAA I saw it. Thanks for clarifying.

**Comment by WenMeng-NOAA** [comment]
> @WenMeng-NOAA looks like one of the checks failed...maybe I did something wrong when syncing my branch with the latest EMC develop...let me know what you recommend...

@EricJames-NOAA I re-run the failed CI test. Now all CI tests look good. I would assume that's github issue. All your changes look good to me.

**Comment by EricJames-NOAA** [comment]
> > @WenMeng-NOAA looks like one of the checks failed...maybe I did something wrong when syncing my branch with the latest EMC develop...let me know what you recommend...
> 
> @EricJames-NOAA I re-run the failed CI test. Now all CI tests look good. I would assume that's github issue. All your changes look good to me.

@WenMeng-NOAA thanks!  Sounds good.

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on WCOSS2, Hera and Orion. The baselines for rap/hrrr and 3drtma will be updated with this PR.

**Comment by WenMeng-NOAA** [comment]
This PR is ready for merging.

**Review by WenMeng-NOAA — APPROVED** [review]

**Pull request merged**